### ISSUE TITLE
Setup server command crash bot

### ISSUE THREAD

**Issue by jcolfej** [circumspect/White-Rabbit#164]
**Description**
Impossible to setup server in french.

```
2023-05-10 17:40:54 INFO     white-rabbit Logging in...
2023-05-10 17:40:54 INFO     discord.client logging in using static token
2023-05-10 17:41:06 INFO     discord.gateway Shard ID None has connected to Gateway (Session ID: 409361c51b760e8f1e486e4424ae7717).
2023-05-10 17:41:08 INFO     white-rabbit Bot has logged in!
2023-05-10 17:53:28 ERROR    discord.client Ignoring exception in on_command_error
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "/whiterabbit/./src/__main__.py", line 136, in on_command_error
    bot_channel = utils.get_text_channels(ctx.guild)[BOT_CHANNEL]
KeyError: 'commandes-bot'
```

**Reproducing**
1. Do setup instructions (discord bot, discord server using template, etc...)
2. Send !server_setup

**Expected behavior**
All channels were deleted, but nothing after that.
No french channels created.

**System Info**
Using Docker image in version 0.15.0.

**Comment by circumspect** [comment]
Hi,
It seems like this issue was caused by a missing argument in the workflow file I was using to update docker hub, but it should be fixed now. I've updated both the latest and 0.15.0 tags there, but let me know if you're still having issues!

**Comment by circumspect** [comment]
Is the white rabbit role the highest in the server?

**Comment by circumspect** [comment]
Fixed in 0.16.0
